contracts/modules/uniswap/v3/BytesLib.sol

```diff
@@ -29,27 +29,48 @@ library BytesLib {
         }
     }
 
+    /// @notice Decode the `arg`-th element in `_bytes` as a dynamic array
+    /// @dev The decoding of `length` and `offset` is universal,
+    /// whereas the type declaration of `res` instructs the compiler how to read it.
+    /// @return length Length of the array
+    /// @return offset Pointer to the data part of the array
+    function toLengthOffset(bytes calldata _bytes, uint256 arg)
+        internal
+        pure
+        returns (uint256 length, uint256 offset)
+    {
+        assembly {
+            // The offset of the `arg`-th element is `32 * arg`, which stores the offset of the length pointer.
+            let lengthPtr := add(_bytes.offset, calldataload(add(_bytes.offset, mul(32, arg))))
```

Comment:
We like to consistently use hex for our numbers in assembly - so please can you change `32` to `0x20`

Reply:
Sure. I intentionally used decimals as `toPool` uses decimals.